broadinstitute/single_cell_portal_core
# Adding SENTRY_AUTH_TOKEN to necessary scripts/configs (SCP-5299)

#### BACKGROUND & CHANGES
This fixes a bug where `SENTRY_AUTH_TOKEN` was not passed into the running portal Docker container.  This token is needed to upload sourcemaps to Sentry to allow un-minifying Javascript when examining errors.

#### MANUAL TESTING
1. Boot via Docker with `bin/docker-compose-setup.sh`
2. In the terminal output, note that the `single_cell_vite` service starts without error and you do not see the following message: `[sentry-vite-plugin] Warning: No auth token provided. Will not upload source maps. Please set the 'authToken' option.`
